src/main/scala/gestalt/Symbols.scala

```diff
@@ -15,5 +15,8 @@ trait Symbols { this: Toolbox =>
 
     /** type of a member with respect to a prefix */
     def asSeenFrom(mem: Symbol, prefix: Type): Type
+
+    /** subst symbols in tree */
+    def subst(tree: tpd.Tree)(from: List[Symbol], to: List[Symbol]): tpd.Tree
```

Comment:
Yes, now it's ready for view. I've used the most clumsy way to implement it -- it seems to be  doing typed transform is not very different from compiler.

I ended up using `changeOwner` explicitly, and meta-programmers need to be careful that all typed definitions are created using the `toolbox.owner` as parent.